heracles/dices/polspice.py

```diff
@@ -0,0 +1,200 @@
+# DICES: Euclid code for harmonic-space statistics on the sphere
+#
+# Copyright (C) 2023-2024 Euclid Science Ground Segment
+#
+# This file is part of DICES.
+#
+# DICES is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# DICES is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with DICES. If not, see <https://www.gnu.org/licenses/>.
+#####
+# WIP
+#####
+import numpy as np
+from scipy.special import lpn as legendrep
```

Comment:
Can we remove the dependency on `scipy` here with the functions from numpy?

Reply:
I would advice we don't since this is the one we have tested against the CAMB implementation